fast/stages/0-bootstrap/organization.tf

```diff
@@ -58,23 +58,52 @@ locals {
     "tag_viewer",
     "tenant_network_admin",
   ]
-  group_iam = {
+  # intermediate values before we merge in what comes from the checklist
+  _group_iam = {
     for k, v in local.iam_group_bindings : k => v.authoritative
   }
-  iam = merge(
+  _iam = merge(
     {
       for r in local.iam_delete_roles : r => []
     },
     {
       for b in local._iam_bindings_auth : b.role => b.member...
     }
   )
-  iam_bindings_additive = {
+  _iam_bindings_additive = {
     for b in local._iam_bindings_add : "${b.role}-${b.member}" => {
       member = b.member
       role   = b.role
     }
   }
+  # final values combining all sources
+  group_iam = {
+    for k, v in local._group_iam : k => distinct(concat(
+      v,
+      try(local.checklist.group_iam[k], [])
+    ))
+  }
+  iam = {
+    for k, v in local._iam : k => distinct(concat(
+      v,
+      try(local.checklist.iam[k].authoritative, [])
+    ))
+  }
+  iam_bindings_additive = merge(
+    local._iam_bindings_additive,
+    {
+      for k, v in try(local.checklist.iam_bindings, []) :
+      v.key => v if lookup(local._iam_bindings_additive, v.key, null) == null
```

Comment:
This only adds permission, if there is no same key in variables. Given the construction of the keys, this should be exactly the same permission, but why not `disctinct(concat(...))` them here too just in case?